styles/css/vtex.store-header.css

```diff
@@ -1,18 +1,66 @@
-.headerRow--menu-link :global(.vtex-menu-2-x-styledLink) {
-  color: #ffffff;
-  font-size: 14px;
+/* add transitions */
+:global(.vtex-sticky-layout-0-x-container) .logoLink,
+:global(.vtex-sticky-layout-0-x-container)
+  :global(.vtex-minicart-2-x-openIconContainer),
+:global(.vtex-sticky-layout-0-x-container)
+  :global(.vtex-store-components-3-x-searchBarContainer) {
+  transition: all 0.3s ease;
+  display: block;
 }
 
-.headerRow--main-header .headerRowBackground {
-  background-color: #F0F0F0;
+/* desktop/mobile main header padding and background transitions */
+:global(.vtex-flex-layout-0-x-flexRowContent--main-header),
+:global(.vtex-flex-layout-0-x-flexRowContent--main-header-mobile) {
+  transition: background 0.3s ease, box-shadow 0.3s ease;
```

Comment:
por mim eu desabilitava essas transiçoes, pq sao transiçoes nao performaticas e nao queria incentivar as pessoas a fazerem isso. mas fica a teu criterio

Reply:
era just for show, posso tirar sim